polling_stations/apps/data_importers/management/commands/import_worthing.py

```diff
@@ -1,29 +1,31 @@
-from django.contrib.gis.geos import Point
 from data_importers.management.commands import BaseXpressDemocracyClubCsvImporter
 
 
 class Command(BaseXpressDemocracyClubCsvImporter):
-    council_id = "E07000229"
-    addresses_name = "parl.2019-12-12/Version 1/merged.tsv"
-    stations_name = "parl.2019-12-12/Version 1/merged.tsv"
-    elections = ["parl.2019-12-12"]
+    council_id = "WOT"
+    addresses_name = "2021-02-15T11:21:49.443491/WBCDemocracy_Club__06May2021.tsv"
+    stations_name = "2021-02-15T11:21:49.443491/WBCDemocracy_Club__06May2021.tsv"
+    elections = ["2021-05-06"]
     csv_delimiter = "\t"
-    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
-        rec = super().station_record_to_dict(record)
 
-        # user error report #205
-        # Richmond Room (adj. to Assembly Hall)
-        if record.polling_place_id == "1837":
-            rec["location"] = Point(-0.372557, 50.814974, srid=4326)
-
-        return rec
-
-    def address_record_to_dict(self, record):
-        rec = super().address_record_to_dict(record)
-
-        if record.addressline6 in ["BN11 2FL", "BN11 2FJ", "BN11 1BF"]:
-            rec["accept_suggestion"] = False
-
-        return rec
+        # Moved to correct address => Richmond Room (adj. to Assembly Hall), Stoke Abbott Road, Worthing
+        if record.polling_place_id == "2171":
+            record = record._replace(polling_place_easting="514729")
+            record = record._replace(polling_place_northing="102975")
+
+        # Improved accuracy of station locations
+        if (
+            record.polling_place_id == "2143"
+        ):  # St. Richard's Church, Collingwood Road, Worthing
+            record = record._replace(polling_place_easting="511512")
+            record = record._replace(polling_place_northing="103575")
+
+        if (
+            record.polling_place_id == "2155"
+        ):  # Palatine Park Football Centre, Palatine Road, Worthing
+            record = record._replace(polling_place_easting="511285")
+            record = record._replace(polling_place_northing="103912")
+
+        return super().station_record_to_dict(record)
```

Comment:
Here I think you're more likely to be right about the correction, but I'd still leave it out for reasons out lined above. The reason for leaving the correction to assembly rooms in is a. it's from a previous script which references a user report; and b. it's a much bigger change, so will impact on users/is more 'obviously wrong.